demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -865,6 +865,46 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
     );
   }
 
+
+  async getStatsForOutbound(id: string): Promise<void> {
+    const videoElement = document.getElementById(id) as HTMLVideoElement;
+    const stream = videoElement.srcObject as MediaStream;
+    const track = stream.getVideoTracks()[0];
+    let basicReports: {[id: string]: number} =  {};
```

Comment:
nit: `basicReports` and `reports` can just be `const` as you just add properties.